### DISCUSSION TITLE
Missing Filters on Collection (limited to 100?)

### DISCUSSION THREAD

**Discussion by portedison** [Shopify/storefront-api-feedback#147] [Help]
@johncraigcole 

Is there a limit to the filter values returned in collection product filters? We're using the filters for lists of designers etc. But are missing some  from the list, the max items returned per filter seems to be 100. This isn't documented anywhere, if this is the case can this restriction be extended or paginated?

```collection(handle: $handle) {
      products(
        filters: $filters
      ) {
        ...productConnection
        filters {
            values {
                [this seems to be limited to 100?]
            }
        }
        ...
```

**Comment by johncraigcole** [comment]
@portedison at first pass your experience of a limit of 100 appears correct. Let me check if there's any plans to make the filters array a pageable set. At a minimum, I'll look at getting the documentation to reflect the limit of 100.

**Reply by portedison** [reply]
Thanks for looking into this @johncraigcole. We've detected another issue related to the filters. If I do the following two request, when using the collection endpoint (a collection matching all 5000 odd products in our store), I only get a subset of results.

This request is returning a subset (4 items).
```
query getCollection {
  collection(handle: "all-products") {
    products(first: 100, filters: [{productVendor: "Brokis"}]) {
      nodes {
        ...

```

This request is returning all correct items (21 items).
```
query getAllProducts {
    products(first: 100, query: "vendor:brokis*") {
      nodes {
         ...
```
